client/src/App.css

```diff
@@ -1,9 +1,42 @@
 .body {
-
   background-color: powderblue;
 }
 
+.question {
```

Comment:
I recommend to use Globalsyles (Styled Components) instead of mixing. It is possible, but not good practice. Maybe you can think about this later  🐜